### DISCUSSION TITLE
Delete individual commands in command queue?

### DISCUSSION THREAD

**Discussion by jgiebler** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#518] [General]
Is there a way to delete individual commands? I have multiple repeating commands. I would like to "edit" (remove and re-add) just one of them. Is there a way to do this without clearing the whole queue?

**Comment by visualapproach** [comment]
It's not "possible" atm. I think you can download the cmdq.json or .txt and edit and upload again, but that may take more effort than rewriting the queue.

**Comment by jgiebler** [comment]
That's great info! I didn't even think about pulling down the file and editing. That will work. Thanks for such quick feedback!

**Reply by visualapproach** [reply]
> That's great info! I didn't even think about pulling down the file and editing. That will work. Thanks for such quick feedback!

You deserve it mate 🤩

**Reply by visualapproach** [reply]
When I came to think of it, You would need to hard reset (pressing reboot button) after you've modified and saved your file, otherwise it will be written over.
